src/models/db/account.js

```diff
@@ -37,6 +37,8 @@ class Account extends Model {
     if (data.id) {
       data.name = uri.parse(data.id, 'account').name.toLowerCase()
       delete data.id
+    } else {
+      data.name = data.name.toLowerCase()
```

Comment:
Can you add a test case for PUTing an account that is missing both `id` and `name`? We need to make sure it isn't a 500 error.